src/main/java/org/opentripplanner/graph_builder/module/osm/OsmModule.java

```diff
@@ -55,6 +57,8 @@ public class OsmModule implements GraphBuilderModule {
   private final SafetyValueNormalizer normalizer;
   private final VertexGenerator vertexGenerator;
   private final OsmDatabase osmdb;
+  private ImmutableTable<String, String, Map<MobilityProfile, Float>> mobilityProfileData;
+  private HashMap<String, Boolean> mappedMobilityProfileEntries;
```

Comment:
If you only want to keep tabs of those edges that have been matched you can simply use a `HashSet`, which saves a bit of memory because you don't need to create a extra wrapper object per match.

Reply:
Good idea, refactored in https://github.com/ibi-group/OpenTripPlanner/pull/197/commits/64d521f8fa59570666fcd53eec629db5a5699fd4.